.buildkite/staging-tests.json

```diff
@@ -24,7 +24,7 @@
     },
     {
       "test_name": "staging: unittests-gnu",
-      "command": "cd staging && cargo test --all-features --workspace",
+      "command": "cd staging && export ALSA_CONFIG_PATH=$(pwd)/alsa.conf && cargo test --all-features --workspace",
```

Comment:
Can we move this `env` in the test as we do for PW in `PipewireTestHarness::new()`?

Reply:
Probably yea. We need it once for every test thread. I will do it tomorrow!

Reply:
sure, no rush ;-)